src/strategies/erc721-and-erc1155-flattened/index.ts

```diff
@@ -0,0 +1,85 @@
+import { multicall } from '../../utils';
+import snapshotjs from '@snapshot-labs/snapshot.js';
+
+export const author = 'hotmanics';
+export const version = '0.1.0';
+
+const abi721 = [
+  'function balanceOf(address account) external view returns (uint256)'
+];
+
+const abi1155 = [
+  'function balanceOf(address _owner, uint256 _id) external view returns (uint256)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const erc1155BlockTag =
+    typeof options.erc1155BlockTag === 'number'
+      ? options.erc1155BlockTag
+      : 'latest';
+
+  const erc721BlockTag =
+    typeof options.erc721BlockTag === 'number'
+      ? options.erc721BlockTag
+      : 'latest';
```

Comment:
to avoid issues, better to use two https://snapshot.org/#/strategy/validation strategies inside voting validation https://docs.snapshot.org/user-guides/strategies/validation-strategies#voting-validation-in-space-settings

Reply:
> to avoid issues, better to use two https://snapshot.org/#/strategy/validation strategies inside voting validation https://docs.snapshot.org/user-guides/strategies/validation-strategies#voting-validation-in-space-settings

Will the validation strategies flatten to 1 vote for X validations? Or is it X votes for X validations?
I am looking to do: if (user.erc721balance >= 1 && user.erc1155balance >= 10) { user.votingPower = 1 }

Reply:
Yep that should work